cpu/cc2538/Makefile.cc2538

```diff
@@ -84,7 +85,7 @@ CUSTOM_RULE_LINK=1
 
 %.elf: $(CPU_STARTFILES) %.co $(PROJECT_OBJECTFILES) $(PROJECT_LIBRARIES) contiki-$(TARGET).a $(LDSCRIPT)
 	$(TRACE_LD)
-	$(Q)$(LD) $(LDFLAGS) ${filter-out $(LDSCRIPT) %.a,$^} ${filter %.a,$^} $(TARGET_LIBFILES) -o $@
+	$(Q)$(LD) $(LDFLAGS) -Wl,--start-group ${filter-out $(LDSCRIPT) %.a,$^} ${filter %.a,$^} $(TARGET_LIBFILES) -lc -Wl,--end-group -o $@
```

Comment:
I have to revisit that one.

You might remember our conversation on the mailing list. It took me quite a while to get the linking with an external library, included with PROJECT_LIBRARIES += working.